org.lflang.ide/build.gradle

```diff
@@ -1,52 +1,7 @@
-import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 dependencies {
     implementation project(':org.lflang')
     implementation "org.eclipse.xtext:org.eclipse.xtext.ide:${xtextVersion}"
     implementation "org.eclipse.xtext:org.eclipse.xtext.xbase.ide:${xtextVersion}"
```

Comment:
What does this gradle file build? Is it the language server?

Reply:
Yes, `ide` contains the code for the xtext-generated language server. Therefore, the `diagram` plugin depends on it.
Anyway, it is only temporary as we will merge all those folders in the next restructuring step.